docs/docsite/rst/getting_started_ee/setup_environment.rst

```diff
@@ -0,0 +1,37 @@
+.. _setting_up_environment:
+
+###########################
+Setting up your environment
+###########################
+
+Complete the following steps to set up a local environment for your first execution environment:
+
+1. Ensure the following packages are installed on your system:
+
+* ``podman`` or ``docker``
+* ``python3``
+
+If you use the DNF package manager, install them as follows:
+
+.. code-block:: bash
+
+  sudo dnf install -y podman python3
+
+2. Upgrade the `pip` package manager and install `ansible-navigator` to run EEs.
```

Comment:
isn't the point of `ansible-navigator` to verify the EEs and test them out? The way it's worded here suggests anyone using EEs has to use navigator to run them.

Reply:
it's a valid way to run EEs in CLI, so i changed `lets you run EEs on the command line`